### PR TITLE
Material Generation Option

### PR DESCRIPTION
**Pull request by defektu**
Simple tweak that let's you generate materials instead of colors.

#Changed Blender render engine to Eevee for testing purposes.

### CODE REVIEW

File: Batch_Json_files/.gitignore [hunk 1]
```diff
@@ -1,4 +0,0 @@
-# This directory is intentionarly ignored - this file only applies to the git repo
```
Comment (torrinworx) [inline Batch_Json_files/.gitignore:1]: Can this file be added back in? It's important for testing and stops any batch.json files from accidentally being uploaded to main

**Review by torrinworx — CHANGES_REQUESTED** [review]
All good, can you just resolve the conflicts with the main branch? I'll approve it after that.

File: Batch_Json_files/.gitignore (hunk 1 above)
Comment (defektu) [reply Batch_Json_files/.gitignore:1]: Oops... Sorry 'bout that. Accidentally deleted when testing.

Comment (torrinworx) [reply Batch_Json_files/.gitignore:1]: All good! Just added it back in another commit

**Pull request merged**